### ISSUE TITLE
Progress indicator confused for radio bottons (interactive to progress)

### ISSUE THREAD

**Issue by meirachefitz** [carbon-design-system/carbon#8104]
## Environment

> Operating system
Any
> Browser
Any

Within a service in IBM Cloud Platform

## Detailed description

> What version of the Carbon Design System are you using?
latest
> What did you expect to happen?
Users expected that if they click on an "empty circle", it would move them to the next step of the process. 
> What happened instead?
Only after they noticed the "radio button" was not clickable did the users see the bottom right button to proceed to the next step.

> What WCAG 2.1 checkpoint does the issue violate?
I could not map this to a specific item in WCAG 2.1 

## Steps to reproduce the issue

Dependent upon access to the service.

> Please create a reduced test case in CodeSandbox
>

## Additional information
![Screen Shot 2021-03-16 at 2 40 40 PM](https://user-images.githubusercontent.com/80786318/111369962-a3a12d00-8665-11eb-89d5-bd472a6e4835.png)
This is where we are seeing the issue

**Comment by tw15egan** [comment]
cc @carbon-design-system/design

**Comment by laurenmrice** [comment]
We plan to update the icons in this component for this reason. It will look something like this but we will post final design specs once we come to a conclusion with our icon indicator guidance that we are working on right now. 👍

![Artboard](https://user-images.githubusercontent.com/43969356/111623832-ea536c00-87b8-11eb-99e8-2cd8f895e8d6.png)

**Comment by tw15egan** [comment]
@laurenmrice has there been any more work on this?

**Comment by laurenmrice** [comment]
@tw15egan Not yet. Once we get some dedicated design time for this we will update the spec. 👍

**Comment by tymekg** [comment]
Hi, not sure if it's worth a separate issue, so commenting here:
@laurenmrice The blue bar on top looks confusing. It does not reinforce the message about progress but adds ambiguity instead. This occurs both in your new design pasted above and in the current implementation (we are using v10.30):
![image](https://user-images.githubusercontent.com/16018298/127157188-b6f218a0-ff73-4f3b-9465-5ae246a404d7.png)
I think the bar is stretched to far, it makes impression that it's almost done with the Running step, while actually the step Pending is still in progress.

**Comment by thyhmdo** [comment]
> We plan to update the icons in this component for this reason. It will look something like this but we will post final design specs once we come to a conclusion with our icon indicator guidance that we are working on right now. 👍
> 
> ![Artboard](https://user-images.githubusercontent.com/43969356/111623832-ea536c00-87b8-11eb-99e8-2cd8f895e8d6.png)

This is correct according to our guidance. Except that we have blue for the Not started. And seems like we don't have that icon in either our Icon page or Sketch. We also don't have a specific icon for disabled, but the description says that it is used for upstarted tasks or disabled processes, I think it makes sense. 




![image](https://user-images.githubusercontent.com/77697351/142691501-ca2fcfdc-f3d2-4012-9460-6b4d3e10ecfc.png)

**Comment by thyhmdo** [comment]
@abbeyhrt the final spec
![image](https://user-images.githubusercontent.com/77697351/145882036-3bf4964b-6a0a-417d-bdf4-c5776eb897f7.png)

**Comment by abbeyhrt** [comment]
@thyhmdo thank you for the final spec! Do we know when the "Not started" icon will be available?

**Comment by laurenmrice** [comment]
@abbeyhrt The "Not started" icon is available. The icon name is just `circle-dash` https://carbon-elements.netlify.app/icons/examples/preview/#CircleDash16
